src/test/java/teammates/test/cases/logic/EmailGeneratorTest.java

```diff
@@ -179,6 +181,28 @@ public void testGenerateFeedbackSessionEmails() throws IOException {
         assertTrue(hasStudent1ReceivedEmail);
         assertTrue(hasInstructor1ReceivedEmail);
         
+        ______TS("feedback session submission email");
+
+        Calendar time = Calendar.getInstance();
+        time.set(Calendar.DATE, 4);
+        time.set(Calendar.MONTH, 8);
+        time.set(Calendar.HOUR_OF_DAY, 5);
+        time.set(Calendar.MINUTE, 30);
+        EmailWrapper email = new EmailGenerator().generateFeedbackSubmissionConfirmationEmailForStudent(
+                session, student1, TimeHelper.formatTime12H(time.getTime()));
+        assertNotNull(email);
```

Comment:
This should not be necessary. `EmailGenerator` functions are not supposed to return any null values.